docs/generated/actions/match.md

```diff
@@ -525,11 +525,9 @@ Please be careful when using this option and ensure the certificates and profile
 If you want to manually decrypt a file you can.
 
 ```no-highlight
-openssl aes-256-cbc -k "<password>" -in "<fileYouWantToDecryptPath>" -out "<decryptedFilePath>" -a -d -md [md5|sha256]
+openssl aes-256-cbc -k "<password>" -in "<fileYouWantToDecryptPath>" -out "<decryptedFilePath>" -a -d
 ```
 
-_**Note:** You may need to swap double quotes `"` for single quotes `'` if your match password contains an exclamation mark `!`._
```

Comment:
This is my bad, I approved & merged this here: https://github.com/fastlane/docs/pull/1196 without noticing that the changes were made on `/generated/` directory 😬 

I'll get this fixed in `fastlane/fastlane`.

Reply:
Fixed here: https://github.com/fastlane/fastlane/pull/21302

Instead of moving forward with this PR as is (causing a regression in the docs), and waiting for the next fastlane deploy to re-generate these docs changes as expected, I'm gonna apply a one-off fix to revert the changes in here, so we can merge these changes, and then next time fastlane gets deployed (assuming it includes https://github.com/fastlane/fastlane/pull/21302 😄 ), everything will be back on track 😃

Reply:
There: https://github.com/fastlane/docs/pull/1197/commits/148ddcb177b98330a48e90d6aa5e2317a30a44a5